src/main/java/DBot.java

```diff
@@ -0,0 +1,111 @@
+import java.util.*;
+
+public class DBot {
+    private static boolean isOn;
+    private static List<Task> datas;
+
+    public static void main(String[] args) {
+        isOn = true;
+        datas = new ArrayList<>();
+
+        String greeting = "____________________________________________________________\n" +
```

Comment:
use "final" and SCREAMING_SNAKE_CASE for the greeting variable as it is a constant